packages/block-library/src/latest-posts/edit.js

```diff
@@ -360,9 +360,9 @@ class LatestPostsEdit extends Component {
 										.trim()
 										.split( ' ', excerptLength )
 										.join( ' ' ) +
-								  ' ... <a href=' +
+								  ' ... <a href="' +
 								  post.link +
-								  'target="_blank" rel="noopener noreferrer">' +
+								  '" target="_blank" rel="noopener noreferrer">' +
 								  __( 'Read more' ) +
 								  '</a>'
```

Comment:
Any reason why we're concatening HTML and using `RawHTML`. security-wise, it seems better to just use JSX.

Reply:
i don't think there is a reason :)

Reply:
@draganescu, can you prioritise a fix?

Reply:
Howdy @youknowriad and @mcsf a fix is ready in #21170